suites/upgrade/infernalis/older/0-cluster/start.yaml

```diff
@@ -0,0 +1,24 @@
+meta:
+   - desc: |
+      white-list items
+      assign cluster nodes
```

Comment:
I don't think it's worth mentioning the whitelist - that's just an implementation detail of the test case.

The way the cluster is setup is significant though - maybe:
desc: 'run ceph on two nodes, using one of them as a client, with a separate client-only node. Use xfs beneath the osds.'